modules/smtp/modules.php

```diff
@@ -28,8 +28,17 @@ public function process() {
                 $this->request->get['uid']
             );
             $reply_details = $this->session->get($cache_name, false);
+
+            $reply_type = '';
             if ($reply_details) {
-                recip_count_check($reply_details['msg_headers'], $this);
+                if (array_key_exists('reply', $this->request->get)) {
```

Comment:
This same thing is repeated in Hm_Handler_load_smtp_servers_from_config::process. Can you make it a function ?